debian.sh

```diff
@@ -265,13 +267,13 @@ debian_grub_fix() {
   MAPPER="$FOLD/hdd/dev/mapper"
   TEMPFILE="$FOLD/hdd/tmp"
 
-  ls -l $MAPPER > $TEMPFILE/tmp.tmp
-  cat $TEMPFILE/tmp.tmp | grep -v "total" | grep -v "crw" | while read line; do
-    VOLGROUP=$(echo $line | cut -d " " -f9)
-    DMDEVICE=$(echo $line | cut -d "/" -f2)
+  ls -l "$MAPPER" > "$TEMPFILE/tmp.tmp"
```

Comment:
possible that this can be removed, I will create a new issue for further analysis